src/app/settings/Spreadsheets.tsx

```diff
@@ -38,44 +41,65 @@ export default function Spreadsheets() {
+            {t('Settings.ExportSS')}
+          </label>
+          <div>
+            <button
+              type="button"
+              className="dim-button"
+              onClick={() => downloadCsv('Weapons')}
+              disabled={disabled}
+            >
+              <AppIcon icon={spreadsheetIcon} /> <span>{t('Bucket.Weapons')}</span>
+            </button>{' '}
+            <button
+              type="button"
+              className="dim-button"
+              onClick={() => downloadCsv('Armor')}
+              disabled={disabled}
+            >
+              <AppIcon icon={spreadsheetIcon} /> <span>{t('Bucket.Armor')}</span>
+            </button>{' '}
+            <button
+              type="button"
+              className="dim-button"
+              onClick={() => downloadCsv('Ghost')}
+              disabled={disabled}
+            >
+              <AppIcon icon={spreadsheetIcon} /> <span>{t('Bucket.Ghost')}</span>
+            </button>
+          </div>
+        </div>
         <div>
-          <button
-            type="button"
-            className="dim-button"
-            onClick={() => downloadCsv('Weapons')}
-            disabled={disabled}
-          >
-            <AppIcon icon={spreadsheetIcon} /> <span>{t('Bucket.Weapons')}</span>
-          </button>{' '}
-          <button
-            type="button"
-            className="dim-button"
-            onClick={() => downloadCsv('Armor')}
-            disabled={disabled}
-          >
-            <AppIcon icon={spreadsheetIcon} /> <span>{t('Bucket.Armor')}</span>
-          </button>{' '}
-          <button
-            type="button"
-            className="dim-button"
-            onClick={() => downloadCsv('Ghost')}
-            disabled={disabled}
-          >
-            <AppIcon icon={spreadsheetIcon} /> <span>{t('Bucket.Ghost')}</span>
-          </button>
+          <FileUpload
+            title={t('Settings.CsvImport')}
+            accept={{ 'text/csv': ['.csv'] }}
+            onDrop={importCsv}
+          />
         </div>
       </div>
-      <div className="setting">
-        <FileUpload
-          title={t('Settings.CsvImport')}
-          accept={{ 'text/csv': ['.csv'] }}
-          onDrop={importCsv}
-        />
-      </div>
+      {d2Defs && (
+        <div className="setting">
+          <div className="horizontal">
+            <label htmlFor="spreadsheetLinks" title={t('Settings.ExportLoadoutSSHelp')}>
+              {t('Settings.ExportLoadoutSS')}
+            </label>
+            <div>
+              <button
+                type="button"
+                className="dim-button"
+                onClick={() => dispatch(downloadLoadoutsCsv())}
+                disabled={disabled}
+              >
+                <AppIcon icon={spreadsheetIcon} /> <span>{t('Loadouts.Loadouts')}</span>
+              </button>
+            </div>
+          </div>
+        </div>
+      )}
```

Comment:
I'm slightly concerned that people could believe they can import the Loadouts CSV - I tried to add some more separation but maybe that's not super clear

![grafik](https://github.com/DestinyItemManager/DIM/assets/14299449/ba772f7a-ab7b-4a8f-b05a-db80f0a1a5a8)

Reply:
I've always meant to move these out of Settings (the inventory spreadsheets are all available from the Organizer page) since they're another view of the data, not a setting. So the button could go on the loadouts view.

Reply:
Hmm, on the Loadouts page there's an "Import Loadout" button that acceps dim.gg/destinyitemmanager.com loadout shares, that could be similarly confusing

Reply:
Yeah that's fair. It's fine as is.